pkg/server/server.go

```diff
@@ -293,6 +308,15 @@ func (s *Server) OnPodUpdate(oldPod, pod *v1.Pod) {
 func (s *Server) OnPodDelete(pod *v1.Pod) {
 	klog.V(4).Infof("OnPodDelete")
 	s.OnPodUpdate(pod, nil)
+	if multiutils.CheckNodeNameIdentical(s.Hostname, pod.Spec.NodeName) {
+		podIptables := fmt.Sprintf("%s/%s", s.Options.podIptables, pod.UID)
```

Comment:
OK, now I think I understand, this the generated directory! It's got the `--pod-iptables` directory PLUS the pod UID, now I understand with the delete.